app/models/flexible_criterion.rb

```diff
@@ -125,7 +125,7 @@ def add_tas(ta_array)
   end
 
   def get_name
```

Comment:
@GitGeminiano It seems this method may have been used earlier to abstract the different attribute names, but is not redundant. Replace usages of `get_name` with `name`, and delete this method from the two criterion classes.